crates/test-files/fixtures/ingots/imported_constant/src/main.fe

```diff
@@ -0,0 +1,8 @@
+use constants::MAX_ITEMS
+
+contract Foo {
```

Comment:
This would be nice to use the new `#test` support but I noticed ingot tests aren't yet supported and I didn't feel like diving into it.

Reply:
hmm. I should implement this

Reply:
I have that working locally but it needs further cleanup.

Reply:
Opened a PR with what I have https://github.com/ethereum/fe/pull/904